execution/types/log.go

```diff
@@ -100,8 +100,9 @@ func (logs Logs) Copy() Logs {
 // ToRPCTransactionLog converts types.Log in a RPCLog.
 func ToRPCTransactionLog(log *Log, header *Header, txHash common.Hash, txIndex uint64) *RPCLog {
 	return &RPCLog{
-		Log:            *log,
-		BlockTimestamp: header.Time,
+		Log: *log,
+		// Arbitrum has not yet inherited this change from geth.
```

Comment:
how this affects us? why not keep?

Reply:
our `GetLogs` result differs in 1 line for every logs returned. I think when arbitrum adds this we an ucomment as well